Source/WebCore/Modules/webauthn/fido/Pin.cpp

```diff
@@ -291,6 +352,15 @@ Vector<uint8_t> encodeAsCBOR(const TokenRequest& request)
     });
 }
 
+Vector<uint8_t> encodeAsCBOR(const SetPinRequest& request)
+{
+    return encodePinCommand(Subcommand::kSetPin, [coseKey = WTFMove(request.m_coseKey), encryptedPin = request.m_newPinEnc, pinUvAuthParam = request.m_pinUvAuthParam] (CBORValue::MapValue* map) mutable {
```

Comment:
I think we should either unmark `request.m_coseKey` as mutable and not move all the members here (can capture by reference instead) or unmark this `encodeAsCBOR` as const, take in an rvalue, and move all the members.